Dockerfile

```diff
@@ -2,4 +2,5 @@ FROM python:2.7
 WORKDIR /code
 ADD . /code
 RUN python setup.py develop
+RUN pip install cliquet[postgresql]
```

Comment:
monitoring came already with ``setup.py develop`` since cliquet is supposed to already have it in setup.